### ISSUE TITLE
Running `SHOW STATS` kills influxd

### ISSUE THREAD

**Issue by mjdesa** [influxdb/influxdb#3126]
On OSX with influx version 0.9 built from source

```
curl -G http://localhost:8086/query --data-urlencode "db=mydb" --data-urlencode "q=SHOW STATS”
```
kills influxd

**Comment by beckettsean** [comment]
@otoolep any ideas when `SHOW STATS` will get wired up again? We can remove it from the docs if it will not be fixed in 0.9.1

**Comment by otoolep** [comment]
It will not be done in 0.9.1 Possibly 0.9.2.

**Comment by tcompart** [comment]
I am totally confused. First '''SHOW DIAGNOSTICES''' was removed and with 0.9.1 '''SHOW STATS'''?? What is this all about? So I should disable my monitoring for the next weeks then?

**Comment by otoolep** [comment]
@tcompart -- the design was significantly changed for the 0.9.0 release, so stats and diagnostics is being re-implemented. 

It sounds like you were making use of the output of these commands? Can you share more about what you were doing? Any feedback on how this functionality can be improved during the re-implementation would be valuable.

**Comment by tcompart** [comment]
I actually need two things regarding my monitoring. First of all I need a check if the machine is still responding. For this I need just a simple text response like "OK". Second I require a check for the cluster state: how many shards, replicas, general cluster state like green yellow red and if the memory exceeds limits to add another machine if needed.

Currently I have to use /ping with disabled authentication and with a hack to f5. The second part does not exist yet ;)

**Comment by dlfryar** [comment]
@otoolep we are (were) using the stats with a Monasca plugin to get monitoring and health data from Influx.  That payload was very useful.

Here's a link to the plugin we have for Monasca to monitor Influx.

https://review.openstack.org/#/c/196167/

**Comment by otoolep** [comment]
OK, thanks.

Unfortunately stats and diagnostics output format will change significantly once re-implemented, because the underlying design has changed significantly.

**Comment by toddboom** [comment]
@otoolep for now, can we just make sure this doesn't panic the server? we'll revisit stats and diags in `v0.9.4`

**Comment by otoolep** [comment]
But it doesn't panic the server @toddboom:

```
$ curl -G http://localhost:8086/query --data-urlencode "q=SHOW STATS"
{"results":[{"error":"SHOW STATS is not implemented yet"}]}
```

**Comment by otoolep** [comment]
We know `SHOW STATS` needs to be re-implemented, and hope to do so in 0.9.4.

**Comment by toddboom** [comment]
:+1:
